gpkitmodels/GP/aircraft/prop/propeller.py

```diff
@@ -0,0 +1,60 @@
+" propeller model "
+from numpy import pi
+from gpkit import Model, parse_variables, SignomialsEnabled, SignomialEquality
+
+class Propeller(Model):
+    """ Propeller Model
+
+    Variables
+    ---------
+    R          10           [m]         prop radius
+    W          10           [lbf]        prop weight
```

Comment:
I don't like the propeller having a fixed weight.  Does it add 10 lbs to the solar model too? I would probably leave it without weight variable for now.  Maybe it can scale with radius?

Reply:
Agreed fixed weights are not correct, but I'm not sure the best simple weight model yet.  I think we should leave it with a small weight value as a placeholder until we can implement a real weight model.  This lets it be added to the aircraft.components list for integration into the overall model without adding an inaccurate weight estimate.